proposals/physical-cameras/README.md

```diff
@@ -0,0 +1,388 @@
+
+    In addition to transform properties, the base camera includes attributes 
+    describing the aperture at the front of the body. The camera body currently 
+    models the aperture and shutter as a non-interchangeable component.
+
+    The aperture describes both the shape of the pupil of the aperture and also 
+    the shutter, and the distance of the physical aperture from the film back.
+
+    Currently the shape of the aperture is modeled as a straight sided polygon.
+    In the future, an arbitrary mask could be specified to use in convolution
+    or raytracing.
+
+    shutter angle is the shutter duration * frame rate * 360 degrees, and not
+    computed here.
+
+     """
+)
+{
+    float        flange_distance = 52.0 (
+        doc = '''distance from film back to lens flange, in mm'''
+    )
+    float         shutter_open = 0.0 (
+        doc = '''offset in seconds from start of exposure'''
+    )
+    float         shutter_duration = 0.02 (
+        doc = '''duration of exposure, in seconds, the default is 1/50th of 
+                 a second'''
+    )
+    float iris = 6.25 (
+        doc = '''the default aperture is 6.25mm, corresponding to f8 for a 
+                 50mm lens'''
+    )
+}
+
+class PhysicalCameraSensor "PhysicalCameraSensor" (
+    doc="""
+    The PhysicalCameraSensor schema describes the film back of the camera.  The
+    film back is the sensor that is exposed to light, and is the primary
+    determinant of the field of view of the camera, by describing the geometry 
+    of the plane where an image is to be resolved.
+
+    Since a camera may have multiple film backs, the PhysicalCameraSensor is
+    a multi-apply schema, and may be applied multiple times to a PhysicalCamera.
+
+    The sensor's coordinate system has its center at (0, 0) and its bounds 
+    are -1 to 1.
+    The enlarge property is a multiplicative value; and shift is an additive 
+    value in the sensor's coordinate system.
+
+    Shift and enlarge can be used to create projection matrices for
+    subregions of an image. For example, if the default sensor is to be
+    rendered as four tiles, a matrix for rendering the upper left quadrant
+    can be computed by setting enlarge to { 2, 2 }, and
+    shift to millimeters{-17.5f}, millimeters{-12.25f}.
+
+    The default sensor aperture is as for 35mm DSLR.
+
+    A handedness of -1 indicates a left handed camera.
+
+    The PhysicalCameraSensor does describe the electrical or 
+    photochemical characteristics of a sensor, it's geometric.
+    The electrical or photochemical characteristics are to be described
+    either here or in another class within this schema.
+    """
+)
+{
+    float2 shift = (0.0, 0.0) (
+        doc = '''shift in millimeters'''
+    )
+    float2 enlarge = (1.0, 1.0) (
+        doc = '''enlarge multiplicative'''
+    )
+    float2 aperture = (36.0, 24.0) (
+        doc = '''aperture in millimeters'''
+    )
+    float handedness = 1.0 (
```

Comment:
Why a float type for this value?

Reply:
That follows GfMatrix where we see

```

    /// Returns the sign of the determinant of the matrix, i.e. 1 for a
    /// right-handed matrix, -1 for a left-handed matrix, and 0 for a
    /// singular matrix.
    GF_API
    double GetHandedness() const;

    /// Returns true if the vectors in the matrix form a right-handed
    /// coordinate system.
    bool IsRightHanded() const {
        return GetHandedness() == 1.0;
    }

    /// Returns true if the vectors in matrix form a left-handed
    /// coordinate system.
    bool IsLeftHanded() const {
        return GetHandedness() == -1.0;
    }
```

we find code like this throughout OpenUSD:
```
        // negate the angles
        *thetaSw *= -handedness;
        *thetaFB *= -handedness;
        *thetaTw *= -handedness;
```

usd files have it as a token

```
        uniform token orientation = "leftHanded" (
            hidden = true
        )
```

I agree that instead of a handedness float, we should match the `orientation` token usage.